tools/cli/admin_task_list_commands_test.go

```diff
@@ -261,6 +262,136 @@ func TestAdminListTaskList(t *testing.T) {
 	}
 }
 
+func TestAdminUpdateTaskListPartitionConfig(t *testing.T) {
+	// Define table of test cases
+	tests := []struct {
+		name               string
+		setupMocks         func(*admin.MockClient)
+		expectedError      string
+		domainFlag         string
+		taskListFlag       string
+		taskListType       string
+		numReadPartitions  int
+		numWritePartitions int
+	}{
+		{
+			name: "Success",
+			setupMocks: func(client *admin.MockClient) {
+				client.EXPECT().
+					UpdateTaskListPartitionConfig(gomock.Any(), gomock.Any()).
+					Return(&types.UpdateTaskListPartitionConfigResponse{}, nil).
+					Times(1)
+			},
+			expectedError:      "",
+			domainFlag:         "test-domain",
+			taskListFlag:       "test-tasklist",
+			taskListType:       "decision",
+			numReadPartitions:  2,
+			numWritePartitions: 2,
+		},
+		{
+			name: "UpdateTaskListPartitionConfigFails",
+			setupMocks: func(client *admin.MockClient) {
+				client.EXPECT().
+					UpdateTaskListPartitionConfig(gomock.Any(), gomock.Any()).
```

Comment:
For the success case it makes sense to validate the partition config passed to `UpdateTaskListPartitionConfig`